obsei/preprocessor/text_cleaner.py

```diff
@@ -0,0 +1,177 @@
+import logging
+from typing import Any, Dict, List, Optional
+import string
+import re
+from unicodedata import normalize as unicode_decode
+from dateutil.parser import parse
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer
+from nltk.tokenize import word_tokenize
+
+from pydantic import PrivateAttr
+
+from obsei.analyzer.base_analyzer import AnalyzerRequest
+from obsei.preprocessor.base_text_cleaner import (
+    BaseTextCleaner,
+    BaseTextCleanerConfig,
+)
+
+logger = logging.getLogger(__name__)
+nltk.download("punkt")
```

Comment:
Sorry for bringing it again but this global statement execute whenever user import this file even though user does not want to initialise any class in this file (Like adding import in __init__.py).

Reply:
I think it is better to add another class `class TextCleanerConfig(BaseTextCleanerConfig)`

Reply:
@lalitpagaria I'm unable to wrap my head around how would additional class resolve this? Could you please elaborate, also what do you think of the following:
```python
try:
    nltk.data.find("tokenizers/punkt")
except LookupError:
    nltk.download("punkt")
```